server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,18 @@
+import Service from "../../models/Service.js"
+import User from "../../models/User.js"
+
+class ReviewSeeder {
+  static async seed(){
+    const james = await User.query().findById(1)
+    const nana = await User.query().findById(2)
+    const nicksCo = await Service.query().findById(1)
+    const lillysCo = await Service.query().findById(2)
```

Comment:
Unfortunately, it is better (JUST IN THE SEEDER) to find these services not by their ids, but by some specific characteristic, in the off chance that the seeders run poorly. 

Otherwise, you could grab these users and services, but then you would want an `if` statement that says we wont' create a review unless the user and service query came back with something. 

so either find these by property, or update your related queries to run if say the james and nicksCo queries are successful